zulipterminal/config/symbols.py

```diff
@@ -86,3 +86,9 @@
     bline=_MESSAGE_RECIPIENTS_BOTTOM,
     brcorner=_MESSAGE_RECIPIENTS_BOTTOM,
 )
+
+LEFT_ARROW = "←"  # LEFTWARDS ARROW, U+2190
+UP_ARROW = "↑"  # UPWARDS ARROW, U+2191
+RIGHT_ARROW = "→"  # RIGHTWARDS ARROW, U+2192
+DOWN_ARROW = "↓"  # DOWNWARDS ARROW, U+2193
+ARROW_SYMBOLS = [LEFT_ARROW, UP_ARROW, RIGHT_ARROW, DOWN_ARROW]
```

Comment:
For convenient use in lint-hotkeys to check if the key_combination contains any of the arrow keys.